deployment/chatwoot-worker.1.service

```diff
@@ -7,7 +7,7 @@ Type=simple
 User=chatwoot
 WorkingDirectory=/home/chatwoot/chatwoot
 
-ExecStart=/bin/bash -lc 'bundle exec sidekiq -C config/sidekiq.yml'
+ExecStart=/bin/bash -lc 'dotenv bundle exec sidekiq -C config/sidekiq.yml'
```

Comment:
@vishnu-narayanan, since we load environment variables in Linux installations using dotenv. we will need to have this. Could you access the impact? also, see if we need to update any documentation.

Reply:
@sojan-official LGTM